bugwarrior/services/gitea.py

```diff
@@ -524,10 +522,38 @@ def issues(self):
                     self.get_owned_repo_issues(
                         self.username + '/' + repo)
                 )
-            issues.update(
-                filter(self.filter_issues,
-                       self.get_directly_assigned_issues(self.username).items())
-            )
+
+            '''
+            A variable used to represent the attachable HTTP query that can be attached to the /repos/issues/search API end.
+
+            if httpQuery is set to "review_requested=True?mentioned=True" for example, then the /repos/issues/search API end will be told to search for all issues where a review is requested AND where the user is mentioned.
+            '''
+            httpQuery = "limit=" + str(self.config.issue_limit) + "&"
+
+            if self.config.get('include_assigned_issues', True, bool):
```

Comment:
While I questioned whether these options are necessary above, I would note that the way you're accessing the configuration here and below doesn't make a lot of sense to me. You are defining the default values as `False` in the `GiteaConfig` class.

```py
include_assigned_issues: bool = False
```

But here you change the effective default to be `True`. It would make more sense to set the desired default in `GiteaConfig` and access is simply with `self.config.include_assigned_issues`.